pages/submissionFile/submissionFile.js

```diff
@@ -0,0 +1,76 @@
+// @ts-check
+const express = require('express');
+const asyncHandler = require('express-async-handler');
+const { isBinaryFile } = require('isbinaryfile');
+const mime = require('mime');
+const sqldb = require('@prairielearn/postgres');
+
+const sql = sqldb.loadSqlEquiv(__filename);
+const router = express.Router({ mergeParams: true });
+
+const MEDIA_PREFIXES = ['image/', 'audio/', 'video/'];
```

Comment:
I think it may be a good idea to include `application/pdf`, even if `pl-file-preview` (and other elements) may not yet support it.

Reply:
Added in 703283f.